### PR TITLE
Fixed #155 - Add 'Comment Template' to comment logic block.

### LINKED ISSUE #155: Add 'Comment Template' to comment logic block.
Add **'Comment Template'** to comment logic block.

_[Original]_
- Only comment beginning with # is provided.

_[Improved]_
- Provides **'comment template'** functionality applicable to Python Class and Method.
- You can add a new template easily.
- **'Comment Templates'** are referenced in the **Numpy Style Guide**.
- [Reference Document 1](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy-style-python-docstrings)
- [Reference Document 2](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)

_Why Numpy Style Guide?_
- Visual Python has many features for Data Analysis and Machine Learning.
- The Numpy Library is most often used in Data Analysis and ML.

A new function is included in .gif image.

comment_template_clip

I want feedback.

thank you.

### PR DESCRIPTION
**Pull request by Minku-Koo**
- Add comment template on logic block
- Templates are consisting of default, class and method

### CODE REVIEW

**Comment by Minku-Koo** [comment]
Add 'Comment Template'

File: visualpython/js/m_logic/Comment.js
Comment (minjk-bl) [inline visualpython/js/m_logic/Comment.js:1]: It works well :)

**Review by minjk-bl — CHANGES_REQUESTED** [review]
Hello, thank you for the changes you've made, but your repository is not rebased with the current commits. 

You should first fetch & rebase your local repo using 
- `git fetch upstream`  
- `git rebase upstream/devops`

Then push it to your forked-repository using
- `git push origin devops`

In this pull-request, many past-codes are remaining yet. Merging `Comment.js` file is only needed for your request. 

* After you do fetch -> rebase -> push, then let us know to review this PR again. Thanks :)

**Comment by Minku-Koo** [comment]
Thank you for your help.
I create new PR, so close this PR.

**Pull request closed without merging**